packages/ckeditor5-bookmark/ckeditor5-metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+	"plugins": [
+		{
+			"name": "Bookmark",
+			"className": "Bookmark",
+			"description": "Allows to insert a bookmark into the content. // TODO",
```

Comment:
![](https://avatars.githubusercontent.com/godai78?size=32)

[@godai78](https://github.com/godai78) Could you take a look at this description. please?

Reply:
Looking at it.

Reply:
Thank you @godai78